stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -692,6 +741,7 @@
     ;; must be called directly by the tx-sender or by an allowed contract-caller
     (asserts! (check-caller-allowed)
               (err ERR_STACKING_PERMISSION_DENIED))
+    (try! (verify-signer-key-sig pox-addr signer-sig signer-key))
```

Comment:
You'd want to pass in the given `reward-cycle` here

Reply:
To maintain the behavior of "signer key signatures must use _current_ reward cycle", I've passed in `(- reward-cycle u1)` into the signature validation function